src/lib/serviceRoutinesV2/getRegistrations.cpp

```diff
@@ -87,7 +87,7 @@ std::string getRegistrations
     ciP->httpHeaderValue.push_back(double2string(count));
   }
 
-  TIMED_RENDER(out = vectorToJson(registrationV));
+    TIMED_RENDER(out = vectorToJson(registrationV));
```

Comment:
Please avoid breaking existing indentation

```suggestion
  TIMED_RENDER(out = vectorToJson(registrationV));
```

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/3994/commits/397ae9e496a9ce542907b10016aa464b69ba71a3